src/runtime-rs/config/configuration-dragonball.toml.in

```diff
@@ -98,6 +98,12 @@ default_maxvcpus = @DEFMAXVCPUS_DB@
 # > 5                --> will be set to 5
 default_bridges = @DEFBRIDGES@
 
+# Enable balloon f_reporting, default false
```

Comment:
Duplicated description, remove `default false`?

Reply:
Add https://github.com/kata-containers/kata-containers/issues/10281 for this.